src/test/common/test_blkdev_sys_block/sys/dev/block/8:0

```diff
@@ -1 +0,0 @@
-../../block/sda
```

Comment:
I think this and the other files are added to the PR by accident. Please remove them.

Reply:
You might can fix this with a rebase